Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -2594,7 +2594,7 @@ bool CStaticFunctionDefinitions::SetPedOnFire(CClientEntity& Entity, bool bOnFir
 {
     if (IS_PED(&Entity))
     {
-        if (!Entity.IsLocalEntity())
+        if (!Entity.IsLocalEntity() && &Entity != GetLocalPlayer())
```

Comment:
Why?  This function should work with local peds too.

Reply:
This is agreed with botder and dutchman

Reply:
Non-local peds can't use this functions on client-side

Reply:
My bad. I misread this check.